app/views/articles/index.html.erb

```diff
@@ -1,8 +1,16 @@
 <h1> The Viking Times </h1>
 
-<% @articles.each do |article|%>
-    <%=  link_to(article_path(article)) do %>
+<%= link_to 'All', articles_path %>
+
+<% @categories.each do |category| %>
+    <%= link_to category.name, articles_path(params: {category: category.name}) %>
+<% end %>
+
+<% @articles.each do |article| %>
+    <p>
+    <%= link_to(article_path(article)) do %>
```

Comment:
rewrite this to `link_to article_path(article)`

Reply:
ok